p5js/coding-challenges/solar-system-3d-texturized/solar_system_3d_texturized.js

```diff
@@ -151,17 +181,21 @@ class SolarSystem {
     for(var i=0; i<numObjects; i++){
 
       if(i == 0){
-        this.objects.push(this.generateStar());
+        this.objects.push(this.generateStar(i));
       }else{
-        this.objects.push(this.generatePlanet());
+        this.objects.push(this.generatePlanet(i));
```

Comment:
Unfortunately, it took me a while to realize I wasn't passing in the `i` index variable to `generatePlanet(idx)` ... and thus loss some time debugging this. Alas.